stripe-ui-core/src/test/java/com/stripe/android/uicore/address/TransformAddressToElementTest.kt

```diff
@@ -93,75 +90,44 @@ class TransformAddressToElementTest {
         )
     }
 
-    private suspend fun verifySimpleTextSpecInTextFieldController(
-        textElement: SectionSingleFieldElement,
-        simpleTextSpec: TestTextSpec
-    ) {
-        val actualController = textElement.controller as TextFieldController
-        assertThat(actualController.capitalization).isEqualTo(simpleTextSpec.keyboardCapitalization)
-        assertThat(actualController.keyboardType).isEqualTo(simpleTextSpec.keyboardType)
-        assertThat(actualController.label.first()).isEqualTo(
-            simpleTextSpec.label
-        )
-    }
-
     @Test
     fun `Make sure name schema is not found on fields not processed`() {
-        AddressSchemaRegistry.all.forEach { (countryCode, _) ->
-            val schemaList = readFile("src/main/assets/addressinfo/$countryCode.json")
-            val invalidNameType = schemaList?.filter { addressSchema ->
+        AddressSchemaRegistry.all.forEach { (_, schema) ->
+            val invalidNameType = schema.schemaElements().filter { addressSchema ->
                 addressSchema.schema?.nameType != null
+            }.filter {
+                it.type == FieldType.AddressLine1 || it.type == FieldType.AddressLine2
             }
-                ?.filter {
-                    it.type == FieldType.AddressLine1 &&
-                        it.type == FieldType.AddressLine2 &&
-                        it.type == FieldType.Locality
```

Comment:
Does `Locality` still exist?

Reply:
It does but `Locality` can have a name type (`city`, `neighbourhood`, etc...). This tests were never actually checking anything though before because it attempted to do an `AND` comparison on the same field.